app/controllers/desks_controller.rb

```diff
@@ -0,0 +1,41 @@
+class DesksController < ApplicationController
+  before_action :load_desk, only: [:show, :update]
+
+  def index
+    @desks = Desk.all
+  end
+
+  def create
+    desk = Desk.new(desk_params)
```

Comment:
My understanding of our conversation today was that the cloud was responsible for delegating an `id` to the desk (to insure it's always unique). So, I don't think `create` should permit `id`.

Reply:
Right. So the server will return an `id` which the Pi will use to update the status of each desk right?

Reply:
Yup!